eliot/_action.py

```diff
@@ -171,6 +204,7 @@ def __init__(self, logger, task_uuid, task_level, action_type,
                  DeprecationWarning, stacklevel=2)
             task_level = TaskLevel.fromString(task_level)
         self._task_level = task_level
+        self._last_child = None
```

Comment:
:question: An alternate thing you could do is store `_next_child`, and then the type of the attribute stays consistent.